board.py

```diff
@@ -39,3 +40,49 @@ def __setitem__(self, pos: str, item: Piece):
         row = int(pos[1]) - 1
         column = self.columns[pos[0]]
         self.matrix[row][column] = item
+
+    def inside_board(self, i: int, j: int) -> bool:
+        return 0 <= i < 8 and 0 <= j < 8
+
+    def idxs_to_coordinates(self, i: int, j: int) -> str:
+        row = str(i + 1)
+        col = self.inv_col[j]
+        return col + row
+
+    def coordinates_to_idxs(self, coord: str) -> Tuple[int, int]:
+        i, j = int(coord[1]) - 1, self.columns[coord[0]]
+        return (i, j)
+
+    def valid_moves(self, position: str) -> List[str]:
+        curr_piece = self[position]
+        if isinstance(curr_piece, Pawn):
+            return self.pawn_valid_moves(position, curr_piece)
```

Comment:
```suggestion
        curr_piece = self[position]
        moves = []
        if isinstance(curr_piece, Pawn):
            moves = self.pawn_valid_moves(position, curr_piece)
        return sorted(moves)
```

lets sort the moves, so that it will be deterministic for testing the order of the returned moves